### PR TITLE
Remove PMIx man page setup

### PR DESCRIPTION
**Pull request by rhc54**
There are no manpages in v3.2.
Port of https://github.com/openpmix/openpmix/pull/1930

Signed-off-by: Ralph Castain <rhc@pmix.org>
(cherry picked from commit 7b11693783429c43cb30475e4b54e691bf79529c)

### CODE REVIEW

File: opal/mca/pmix/pmix3x/pmix/src/tools/pmix_info/support.c [hunk 1]
```diff
@@ -1327,13 +1327,11 @@ void pmix_info_do_config(bool want_all)
     char *debug;
     char *have_dl;
     char *symbol_visibility;
-    char *manpages;
 
     /* setup the strings that don't require allocations*/
     debug = PMIX_ENABLE_DEBUG ? "yes" : "no";
     have_dl = PMIX_HAVE_PDL_SUPPORT ? "yes" : "no";
     symbol_visibility = PMIX_HAVE_VISIBILITY ? "yes" : "no";
-    manpages = PMIX_ENABLE_MAN_PAGES ? "yes" : "no";
```
Comment (jsquyres) [inline opal/mca/pmix/pmix3x/pmix/src/tools/pmix_info/support.c:1336]: Do you want to just hard-code this output to `no`?

**Review by jsquyres — COMMENTED** [review]
I think you also need to remove the call to `PMIX_SETUP_MAN_PAGES` in `config/pmix.m4`.

File: opal/mca/pmix/pmix3x/pmix/src/tools/pmix_info/support.c (hunk 1 above)
Comment (rhc54) [reply opal/mca/pmix/pmix3x/pmix/src/tools/pmix_info/support.c:1336]: I'd rather not as it implies that one could enable those pages - the fact is that the pages do not exist.

**Comment by rhc54** [comment]
> I think you also need to remove the call to `PMIX_SETUP_MAN_PAGES` in `config/pmix.m4`.

Done

**Review by jsquyres — APPROVED** [review]

**Pull request merged**